apps/locale/ChangeLog

```diff
@@ -20,3 +20,4 @@
       Add second 'dp' argument for decimal places in distance/speed/temp (fix #1523)
 0.17: Fix regression where long month names were 'undefined' (fix #1641)
 0.18: Fix lint warnings, change anv->janv for fr_BE and fr_CH
+    : Deprecate currency information
```

Comment:
Let's pop a version in here

```suggestion
0.19: Deprecate currency information
```

Reply:
The rationale here was that we didn't technically have to release this yet. But I forgot that anything added to this repo automatically becomes the released version. Will fix it properly later (need to change the version in `metadata.json` too).